### ISSUE TITLE
Undefined name 'beta'

### ISSUE THREAD

**Issue by cclauss** [tf-coreml/tf-coreml#81]
flake8 testing of https://github.com/tf-coreml/tf-coreml on Python 2.7.14

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./tfcoreml/optimizations/_optimize.py:116:23: F821 undefined name 'beta'
    beta = np.reshape(beta,(1, shape[0],1,1))
                      ^
```
https://github.com/tf-coreml/tf-coreml/blob/master/tfcoreml/optimizations/_optimize.py#L116

**Comment by aseemw** [comment]
Thank you for pointing out this bug. It is indeed a typo. This will be fixed soon.

**Comment by aseemw** [comment]
PR#84 should resolve this.

**Comment by aseemw** [comment]
Fixed in PR #84
